smoker/client/plugins/example.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2007-2015, GoodData(R) Corporation. All rights reserved
+
+from smoker.client.plugins import SpecificArgument, HostDiscoveryPluginBase
+
+
+class HostDiscoveryPlugin(HostDiscoveryPluginBase):
```

Comment:
example plugin will be discovered and it's arguments added.
I would suggest moving this example to some directory that is not going to be part of the package or better into PyDoc for smoker.client.plugins